main.cpp

```diff
@@ -319,203 +343,272 @@ struct Display
         std::string type = "LVDS";
         int version = 2;
         bool isHighSpeed = true;
+        int currentTransferSpeed = 14400; 
+        bool networkActive = false;
 
         bool isConnected(bool highSpeed = true);
         bool isNetworkConnected();
         std::string getResolution();
-
     };
 
-    // Display text
-    void displayText(DisplayCable myCable);
-    // Display image
-    void displayImage(DisplayCable myCable);
-    // get the active Port
+    void displayText(std::string text);
+    void displayImage(std::string imageFileURL);
     char getActivePort(); 
 
     DisplayCable myCable;
-
 };
 
 
+bool Display::DisplayCable::isConnected(bool highSpeed)
+{
+    return highSpeed && currentTransferSpeed > 10000;
 
+}
 
-/*
-Thing 6) Drumpads
-5 properties:
-    1) Number of Pads
-    2) Pad sensitivity
-    3) Width of a Pad
-    4) Height of Pad
-    5) Pad color Brightness 
-3 things it can do:
-    1) Lit pad
-    2) Check hit pressure
-    3) Default sound bank  
- */
+
+bool Display::DisplayCable::isNetworkConnected()
+{
+    if(networkActive) return true;
+    else return false;
```

Comment:
You didn't change anything.
You don't need the `if/else` since you're just returning the state of `networkActive` 
`return networkActive;`
- [ ] revise